src/components/InputForm.js

```diff
@@ -27,9 +27,9 @@ class InputForm extends React.Component {
       <form>
         <label>
           Ingredients:
-          <input type="text" name = "ing" value={this.props.ing} onChange={this.handleChange} />
-          <button id='add-button' onClick={this.handleChangeAdd}>
-              Add
+          <input type="text" name="ing" value={this.props.ing} onChange={this.handleChange} />
+          <button id='add-ingredient' onClick={this.handleChangeAdd}>
+             Add
           </button>
         </label>
         <br />
```

Comment:
I realise this wasn't you but do we need this break still?

Reply:
No, taken out. 
The two buttons will be in line with the text box